tests/end-to-end/external-solution-integration/internal/scenario/event_mesh/event_mesh.go

```diff
@@ -28,6 +29,7 @@ const (
 
 var (
 	apiRuleRes = schema.GroupVersionResource{Group: "gateway.kyma-project.io", Version: "v1alpha1", Resource: "apirules"}
+	sleep      = func(s *Scenario) time.Duration { return time.Duration(s.waitTime) * time.Second }
```

Comment:
(nit) What is the benefit of assigning it to a variable? We could just have a function named `sleep` 🤔

Reply:
To let the code to be more compact, like in the past. But it's used as a function:
```
testsuite.NewSleep(sleep(s))
```